src/lib/util.ts

```diff
@@ -133,9 +133,9 @@ export function formatTokens(amountWei: BN, symbol?: string, decimals = 18): str
     significantDigits = 1000000000;
     units = "B";
   }
-  else if (tokenAmount.ltn(1000)) {
+  else if (tokenAmount.ltn(100000)) {
```

Comment:
Doesn't this need to be a BN too?

Reply:
`tokenAmount` _is_ a BN

Reply:
Yes but 100000 is not

Reply:
The 1000000 doesn't have to be a BN.  It fits just fine in a `number` and `.ltn()`. takes a number by definition.  The function does have its own smaller limit on the size of the param, but throws an exception if that number is too big, which it isn't.

Reply:
The proof is that the unit tests succeed  :-)

Reply:
ok fine, but then why did you change the other one to new BN(...)?

Reply:
because that one exceeded the maximum value that `ltn()` allows for the input value.